core/src/main/scala/anorm/ParameterMetaData.scala

```diff
@@ -121,7 +121,7 @@ object ParameterMetaData extends JavaTimeParameterMetaData {
   /** Numeric (big integer) parameter meta data */
   implicit object BigIntParameterMetaData extends ParameterMetaData[BigInt] {
     val sqlType  = "NUMERIC"
-    val jdbcType = Types.BIGINT
+    val jdbcType = Types.NUMERIC
```

Comment:
That's specific to Pg, so must not be defined there

Reply:
https://github.com/playframework/anorm/tree/main/postgres/src/main/scala

Reply:
I've removed the change. The problem I have noticed occurs only with `long` types.